code/game/machinery/recharger.dm

```diff
@@ -88,9 +109,10 @@
 			var/obj/item/gun/energy/E = charging
 			if(E.power_supply.charge < E.power_supply.maxcharge)
 				E.power_supply.give(E.power_supply.chargerate)
```

Comment:
Shouldn't this line be modified too for increased recharge rate? Else the recharge just get more and more wasteful for no reason at all.

Reply:
Yeah this is where the multiplication should go